### DISCUSSION TITLE
Audio playback issue at receiver end during audio live streaming after upgrading from 0.11.2 to latest version 0.13.6

### DISCUSSION THREAD

**Discussion by sanjaypanara** [shogo4405/HaishinKit.kt#275] [Q&A]
Here is the link for the audio live streaming issue which is already closed. After your comment in that issue, I have made changes accordingly but audio playback stopped working at receiver end.

https://github.com/shogo4405/HaishinKit.kt/issues/266

We have implemented the changes based on your comments. However, following this update, we are encountering an audio playback issue on the receiver's end during live audio streaming. I do not see any obvious errors on the streaming side.Attached the logs for the streaming side as well for your review here, the receiver is experiencing a SocketTimeoutException while listening.

Is there any other change made in the latest version apart from the encoder? Is there any change related to the streaming side?

Any suggestions to fix this issue? Please let me know.

We have checked again with the older version v0.11.2, in which audio streaming and playback was working fine.

Scenario: I am starting live audio streaming from android device and playback it from another device, but the playback stalls every time, resulting in a SocketTimeoutException. We have also tried playback with iPhone/iPad, and it was not working on those devices either.

Here I am attaching log for streaming end and playback end.
[live audio playback log.txt](https://github.com/shogo4405/HaishinKit.kt/files/15499515/live.audio.playback.log.txt)
[live streaming log.txt](https://github.com/shogo4405/HaishinKit.kt/files/15499517/live.streaming.log.txt)

**Comment by shogo4405** [comment]
Posting the same content in Issues or Discussions won't solve the problem and is a nuisance. Please stop doing it.
